docs/content/Configuration/Config.md

```diff
@@ -434,9 +434,12 @@ with options [from here][link-express-cors-opts].
 
 ### orchestratorOptions
 
-You can pass this object to set advanced options for Cube.js Query Orchestrator.
+<!-- prettier-ignore-start -->
+[[warning | ]]
+| We **strongly** recommend leaving these options set to the defaults. Only **advanced** users should tweak these.
```

Comment:
IDK if this is an actionable and effective advice. How does one decide if they're *advanced enough*? What are the implications of setting these options to unsound values?

My suggestion is that we might replace these with the answer to the second question, and remove the "advanced" thing entirely.

Reply:
That's a great point, @igorlukanin 

```suggestion
<!-- prettier-ignore-start -->
[[warning | ]]
| We **strongly** recommend leaving these options set to the defaults. Changing these values can result in application instability and/or downtime.
```